src/Enterspeed.Source.UmbracoCms/Handlers/EnterspeedJobsHandler.cs

```diff
@@ -68,6 +62,13 @@ public void HandleJobs(IList<EnterspeedJob> jobs)
                 try
                 {
                     handler.Handle(newestJob);
+
+                    // If job has been successfully handled and a failed job exists with same id, we should remove it
```

Comment:
This is a replacement for the "all failed jobs delete". We only delete failed jobs that gets successfully handled

Reply:
Before we had other checks than just the entityId, we also had checks on the culture and the content state

`failedJobsToDelete.AddRange(
                    failedJobsToHandle.Where(
                        x =>
                            x.EntityId == jobInfo.Key.EntityId && x.Culture == jobInfo.Key.Culture && x.ContentState == jobInfo.Key.ContentState));`

Don't know about the content state but we might still need the culture check. We can have the same entity id with multiple cultures, and just because da-dk is ingested doesn't mean that we can delete the e-us e.g.

Reply:
Updated